### ISSUE TITLE
Bug: tsconfig unexpected

### ISSUE THREAD

**Issue by WinnieS0728** [azat-io/eslint-plugin-perfectionist#550]
### Describe the bug

I want to locate my tsconfig.json.

according to [doc](https://perfectionist.dev/rules/sort-imports#tsconfig)

I add tsconfig in 'perfectionist/sort-imports' setting.

but my EsLint keep throwing error about unexpected property.

```ini
[Error - 上午11:58:01] An unexpected error occurred:
[Error - 上午11:58:01] Error: Key "rules": Key "perfectionist/sort-imports":
	Value {"tsconfig":{"rootDir":"."}} should NOT have additional properties.
		Unexpected property "tsconfig". Expected properties: "fallbackSort", "specialCharacters", "ignoreCase", "alphabet", "locales", "order", "type", "customGroups", "maxLineLength", "sortSideEffects", "environment", "tsconfigRootDir", "partitionByComment", "partitionByNewLine", "newlinesBetween", "internalPattern", "groups".
```

using `tsconfigRootDir` will not throw this error. but in doc this is marked as DEPRECATED.
any solution or update?

### Code example

```js
    {
        plugins: {
            perfectionist,
        },
        settings: {
            perfectionist: {
                type: 'line-length',
                order: 'asc',
            },
        },
        rules: {
            'perfectionist/sort-imports': [
                'warn',
                {
                    tsconfig: {
                        rootDir: '.',
                    },
                },
            ],
        },
    },
```

### ESLint version

^9.28.0

### ESLint Plugin Perfectionist version

^4.13.0

### Validations

- [x] Read the [docs](https://perfectionist.dev).
- [x] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by hugop95** [comment]
Hi @WinnieS0728 ,

The feature was added very recently and the docs are a bit in advance, it will be available when @azat-io  creates a release.

**Comment by WinnieS0728** [comment]
thank you.

I will try again after release.🙂

**Comment by azat-io** [comment]
Added in 54b8a42.
Released in [v4.14.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v4.14.0).
